07-bootstrap-validation.md

```diff
@@ -56,18 +56,19 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
 > Mindful Timing for Upgrades:
 > - Be mindful of when upgrades should occur. If you have overlapping maintenance windows, AKS will determine the running order.
 > - To avoid conflicts, leave at least 24 hours between maintenance window configurations. The timing will depend on the number of nodes in your specific cluster and the duration required for upgrades.
+> - Current configuration should not represent a conflict since Kubernetes version updates are applied manually when customer see fit.
 >
 > OS-Level Updates:
-> - By default, the OS-level updates maintenance window is scheduled on a weekly cadence. This is because the OS channel is configured with `NodeImage`, where a new node image is shipped every week.
-> - If you choose the `SecurityPatch` channel, consider changing the maintenance window to daily for more frequent updates.
+> - By default, the OS-level updates maintenance window is scheduled on a daily cadence. This is because the OS channel is configured with `SecurityPatch`, where a new update can be shipped when available.
+> - If you choose the `NodeImage` channel, consider changing the maintenance window to weekly since updates get shipped on a weekly cadence.
 >
 > Kubernetes Version Management:
-> - To stay current with the latest Kubernetes version, a monthly cadence is generally sufficient. However, you can adjust this based on your specific needs.
-> - For more regular updates, configure your cluster to upgrade every two weeks.
+> - To ensure the Kubernetes version is a supported one, a monthly update is generally sufficient. However, it is recommended to track the AKS releases and adjust accordinly.
+> - Being proactive and keep the Kubernetes version current is the best practice.
 >
 > Maintenance Operations:
 > - Keep in mind that performing maintenance operations is considered best-effort. They are not guaranteed to occur within a specific window.
-> - While it’s not strictly recommended, if you require greater control, consider manually updating your cluster.
+> - While it’s not recommended for OS-level updates, if you require greater control, consider manually updating your cluster.
```

Comment:
> While it’s not recommended for OS-level updates, if you require greater control, consider manually updating your cluster.

Which update are you suggesting they consider, it's not clear from this context. OS, node image, cluster, extensions/add-ons?

Reply:
thanks for feedback @ckittel, the following commit should be clearing up most of the notes/details/~guidance we had and being more specific about terms. 

Regarding updates they should consider are AKS node-only updates. 

done | addressed from abe4669d37d515c3ad2e9aa4f9c2d2afe6e85c77